client/payment-methods/index.js

```diff
@@ -128,9 +128,7 @@ const UpeSetupBanner = () => {
 		<>
 			<CardDivider />
 			<CardBody className="payment-methods__express-checkouts">
-				<span className="payment-methods__pill">
-					{ __( 'Early access', 'woocommerce-payments' ) }
-				</span>
+				<Pill>Early access</Pill>
```

Comment:
```suggestion
				<Pill>{ __( 'Early access', 'woocommerce-payments' ) }</Pill>
```

Reply:
D'oh. Thanks!

Reply:
Also fixed the other `Early access` in this file.